src/data/data_fetcher.py

```diff
@@ -153,6 +154,8 @@ def __init__(self, two_year_transaction_period: int, recipient_committee_type: s
         self.api_starting_url_container = _make_api_url(
             two_year_transaction_period, recipient_committee_type
         )
+        self.two_year_transaction_period = two_year_transaction_period
+        self.recipient_committee_type = recipient_committee_type
```

Comment:
Yup, probably a good idea to instantiate your class with the attributes it needs, huh?

My bad, lol 🤦‍♂️

Reply:
It worked just fine without this but to use them outside of `__init__` I had to instantiate.